src/app/domain/components/search-result/search-result.component.scss

```diff
@@ -0,0 +1,78 @@
+.lds-spinner {
+    color: official;
```

Comment:
did u need to add this, isn't there on on design-systems, that you can pull directly?

Reply:
I had seen something here: https://github.com/alphagov/govuk-design-system-backlog/issues/28

Reply:
this has been given to by by Adam Silver I just copped it and pasted it

Reply:
OK, assuming they got plans to add to there library in the future 👍

Reply:
of course